web/src/main/webapp/xslt/skin/default/skin.xsl

```diff
@@ -15,73 +15,91 @@
                   class="navbar-toggle collapsed"
                   data-toggle="collapse"
                   data-target="#navbar"
-                  title="{$i18n/toggleNavigation}"
+                  title="{($i18n/toggleNavigation,'Toggle navigation')[1]}"
```

Comment:
You don't have to do that. There is an issue when running from source with mvn jetty:run but this should not happen after build. 

You probably have log lines like
```
2018-11-04 10:27:16,380 WARN  [geonetwork.search] - Can't find JSON file '/data/dev/gn/dev/web/src/main/webapp/catalog/locales/en-core.json'.
```
with jetty:run ?